demo/configmap.yml

```diff
@@ -7,6 +7,11 @@ metadata:
   namespace: wasm-oidc-plugin
 data:
   envoy.yaml: |
+    admin:
```

Comment:
Enabling the admin interface just for a health check seems a bit excessive to me, since this contains sensitive information. It should of course not be publicly accessible, but disabled altogether seems safest. But I'm not sure, maybe this is standard practice? The manual does suggest using the `/ready` endpoint after all :thinking: 

Another thing I found is that [using the `/ready` endpoint might (in some cases) lead to performance problems.](https://github.com/envoyproxy/envoy/issues/16425) A suggestion I've seen is to [configure a route with a `direct_response` ](https://github.com/envoyproxy/envoy/issues/378#issuecomment-1728846036) for the readiness check, but I haven't verified that it only responds if everything is loaded.

Reply:
i tried `direct_response`, but even if the `/ready` route is configured, it will always go into the plugin and redirect to the auth endpoint there. then i tried setting up a second filter chain, but i complains that the same route configuration is used twice in the same config. 

maybe startup and readiness probes are good for now...